test/conftest.py

```diff
@@ -80,3 +82,26 @@ def pytest_sessionfinish(session, exitstatus):
     # To avoid this, we transform this 5 into a 0 to make testpilot happy.
     if exitstatus == 5:
         session.exitstatus = 0
+
+
+@pytest.fixture(autouse=True)
+def prevent_leaking_rng():
```

Comment:
This is pretty much the same as `freeze_rng_state()` https://github.com/pytorch/vision/blob/main/test/common_utils.py#L86

`autouse=True` means that this fixture will be used by every single test automatically.